pkg/proc/eval.go

```diff
@@ -971,7 +971,15 @@ func (scope *EvalScope) executeOp(stack *evalStack, ops []evalop.Op, curthread T
 		stack.push(newConstant(op.Value, scope.Mem))
 
 	case *evalop.PushLocal:
-		vars, err := scope.Locals(0)
+		var vars []*Variable
+		var err error
+		if op.Frame != 0 {
+			var frameScope *EvalScope
+			frameScope, err = ConvertEvalScope(scope.target, scope.g.ID, int(op.Frame), 0)
```

Comment:
This err here is being ignored and frameScope on the next line could be nil.

Reply:
Fixed.